js/tests/visual/tooltip.html

```diff
@@ -46,9 +46,6 @@
         <button type="button" class="btn btn-secondary" data-toggle="tooltip" data-html="true" title="<em>Tooltip</em> <u>with</u> <b>HTML</b>">
           Tooltip with HTML
         </button>
-        <svg width="30" height="20">
```

Comment:
This SVG is present because it's a visual unit test

Reply:
Which is what I had told you the other day... we need to make this work properly and not on the circle element.

Reply:
I know but by doing that we remove a unit test previously added since a long time

Reply:
Yes, that is why I say, we need to make sure things work with valid stuff and not invalid, random code...

Reply:
Can you open an issue to track that please ?

Reply:
I checked the git log and the original example was added to test an invalid code from the beginning.

So, I'm not sure what to say on the issue, so feel free to make one.